app/io/pathfinder/config/Global.scala

```diff
@@ -0,0 +1,25 @@
+package io.pathfinder.config
+
+import akka.actor.ActorSystem
+import io.pathfinder.routing.Router
+import play.api.GlobalSettings
+import play.api.Application
+import play.Logger
+
+/**
+ * These hooks are called by Play Framework. We can use them to initialize expensive objects and
+ * set up any communication channels.
+ */
+object Global extends GlobalSettings {
+
+    val actorSystem = ActorSystem.create()
+
+    override def onStart(app: Application) {
+        Logger.info("Application has started.")
+        Router.init()
```

Comment:
You can get rid of the `init()` part now. Just use `Router`. Java forces you to use a function.